src/Symfony/Component/Profiler/composer.json

```diff
@@ -0,0 +1,50 @@
+{
+    "name": "symfony/profiler",
+    "type": "library",
+    "description": "Symfony Profiler Component",
+    "keywords": [],
+    "homepage": "http://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Fabien Potencier",
+            "email": "fabien@symfony.com"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "http://symfony.com/contributors"
+        }
+    ],
+    "repositories": [
+        {
+            "type": "git",
+            "url": "git@github.com:jelte/symfony"
+        }
+    ],
```

Comment:
Thank for catching this. been searching for a wile. this will probably fix travis I guess.